lib/teiserver/battle/balance/loser_picks.ex

```diff
@@ -36,6 +36,10 @@ defmodule Teiserver.Battle.Balance.LoserPicks do
   """
   @spec perform([BT.expanded_group_or_pair()], non_neg_integer(), list()) :: BT.algorithm_result()
   def perform(raw_groups, team_count, opts) do
+    # This module doesn't use names or ranks and they need to be dropped from raw_groups to make existing tests pass
+    # However, this module could be improved to use names from raw_groups instead of calling Teiserver.Account
+    raw_groups = Enum.map(raw_groups, fn x-> Map.drop(x,[:ranks, :names]) end)
+
```

Comment:
If you can think of a way to use the names in the input rather than calling Account module that would help. I couldn't quite understand this module.